src/main/java/com/ldtteam/structurize/util/StructureLoadingUtils.java

```diff
@@ -28,7 +32,16 @@
     /**
      * The list of origin folders.
      */
-    public static List<String> originFolders = new ArrayList<>();
+    private static final List<String> originFolders = new ArrayList<>();
+    /**
+     * The list of origin mods.
+     */
+    private static final Map<String, ModFileInfo> originMods = new HashMap<>();
```

Comment:
Theoretically you can get something like mod A adding resources in mod B's namespace (so you'd need to check all mods to know a resource doesn't exist), but I guess you are intentionally ignoring that possibility?

Reply:
Yep, I wondered why you were putting the result from searching through all mods into the modid<->modfile map. You can now completely override mod entry (in this map) by addOriginMod() method. I would like to use resource packs instead (and remove this mess) but I was told that there is some problem with that...